scripts/build_wheel.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+DOCKER_CONDA_ENV="docker_conda_env"
+
+# create conda env and activate it
+conda create -y --force -n ${DOCKER_CONDA_ENV} python=${PYTHON_VERSION} pip
+source activate ${DOCKER_CONDA_ENV}
```

Comment:
use dot instead of source
```suggestion
. activate ${DOCKER_CONDA_ENV}
```